x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -280,4 +280,10 @@ protected boolean enableRoundingDoubleValuesOnAsserting() {
     protected boolean supportsInferenceTestService() {
         return false;
     }
+
+    @Override
+    protected boolean supportsIndexModeLookup() throws IOException {
+        // return hasCapabilities(List.of("join_lookup_v4"));
```

Comment:
CCS does not support lookup-join, and the capabilities are falsely claiming that we have that capability, so we cannot use that here, but must disable lookup-join in CCS IT